core/src/main/java/org/elasticsearch/index/query/SpanMultiTermQueryBuilder.java

```diff
@@ -42,6 +60,42 @@ protected void doXContent(XContentBuilder builder, Params params)
     }
 
     @Override
+    protected Query doToQuery(QueryParseContext parseContext) throws IOException {
+        Query subQuery = multiTermQueryBuilder.toQuery(parseContext);
+        if (subQuery instanceof DateFieldMapper.DateFieldType.LateParsingQuery) {
+            subQuery = ((DateFieldMapper.DateFieldType.LateParsingQuery) subQuery).rewrite(null);
```

Comment:
I was going to say in #12123 that this workaround means SpanMultiTermQuery with this particular type of inner query will not work with `now` then, but I added it here to make our tests work for now. We can track that particular issue in #12123. Wonder if we should add this early rewrite on master as well as I already brought it up there? Need to ask Martjin about it, I guess.